synapse/storage/registration.py

```diff
@@ -351,6 +372,10 @@ def __init__(self, db_conn, hs):
             columns=["creation_ts"],
         )
 
+        self._account_validity_period_s = int(datetime.timedelta(
+            weeks=hs.config.validity_period
+        ).total_seconds())
```

Comment:
Please use timestamp in milliseconds, as that is what we consistently use throughout the code base.